libs/components/tabs/src/lib/modules/tabs/fixtures/tabset-active.component.fixture.html

```diff
@@ -9,7 +9,6 @@
     <sky-tab
       *ngIf="tab3Available"
       [tabHeading]="tab3Heading"
-      [tabHeaderCount]="tab3HeaderCount"
```

Comment:
This was removed as it wasn't being used (`tab3HeaderCount` didn't even exist in the backing TypeScript).